src/Overlay.tsx

```diff
@@ -63,7 +63,7 @@ const StyledOverlay = styled.div<StyledOverlayProps>`
   max-height: ${props => props.maxHeight && heightMap[props.maxHeight]};
   width: ${props => widthMap[props.width || 'auto']};
   border-radius: 12px;
-  overflow: hidden;
+  overflow: visible;
```

Comment:
question: Will this bite us back?

It would be really nice if this _just works_ and doesn't have to be done on case by case basis.

See also, in a completely different context: https://github.com/github/details-dialog-element/issues/82

Reply:
update: It did in fact bite :) removed inline rendering!